docs/products/platform/accounts/guides/service-transfers/index.md

```diff
@@ -1,23 +1,19 @@
 ---
-slug: service-transfers
-description: "Transfer your Linode services to another account conveniently with Service Transfers."
+title: Transfer Services to a Different Account
+title_meta: How to Transfer Services to a Different Linode Account
+description: "Learn how to transfer ownership of Linode services to a different account."
 keywords: ["linode transfer ownership", "transfer linode to another account", "linode can i transfer"]
 tags: ["cloud manager","linode platform","account"]
-license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
-modified: 2021-03-10
+published: 2021-03-10
+modified: 2023-04-04
 modified_by:
   name: Linode
-published: 2021-03-10
 image: TransferOwn_LinodeServ_ST.png
-title: Transferring Ownership of Linode Services with Service Transfers
-title_meta: How to Transfer Ownership of Linode Services
-aliases: ['/guides/platform/manager/service-transfers/']
+aliases: ['/guides/platform/manager/service-transfers/', '/guides/service-transfers/']
 authors: ["Linode"]
 ---
 
-## Linode Service Transfers
-
-The Service Transfer feature in [Cloud Manager](https://cloud.linode.com) can be used to transfer ownership of certain Linode services to another account. Here's how they work:
+Within the Linode Platform, certain services can be transferred from one account to a different account. Here's an overview of this transfer process:
```

Comment:
I think we should declare that Compute instances are transferrable here if they are the only transferrable services at this time.